sdk/storage/azure-storage-blobs/test/ut/bearer_token_test.cpp

```diff
@@ -49,6 +49,18 @@ namespace Azure { namespace Storage { namespace Test {
         clientOptions);
     EXPECT_NO_THROW(blobClient.GetProperties());
 
+    // With custom audience
```

Comment:
You created a separator test case for `BlobContainerClient` and `BlobServiceClient`. Why cannot we do the same for `BlobClient`?

Reply:
I also added a test for `BlobClient`. And I added this test is to test the custom audience when trigger bearer challenge.